src/openrct2-ui/windows/TextInput.cpp

```diff
@@ -202,14 +202,17 @@ class TextInputWindow final : public Window
 
         if (_descriptionStringId == STR_NONE)
         {
-            auto* text = _description.c_str();
+            auto ft = Formatter();
+            ft.Add<const char*>(_description.c_str());
             DrawTextWrapped(
-                &dpi, { windowPos.x + WW / 2, screenCoords.y }, WW, STR_STRING, &text, { colours[1], TextAlignment::CENTRE });
+                &dpi, { windowPos.x + WW / 2, screenCoords.y }, WW, STR_STRING, ft, { colours[1], TextAlignment::CENTRE });
         }
         else
         {
+            auto ft = Formatter();
+            ft.Add<const char*>(TextInputDescriptionArgs);
```

Comment:
This turned out to be incorrect causes #15271. This is a set of arguments and this ends up passing a pointer to the arguments as the arguments which turns in to nonsense. Fixed by https://github.com/OpenRCT2/OpenRCT2/pull/15272